dotCMS/src/test/java/com/dotmarketing/business/cache/provider/CacheSizingUtilTest.java

```diff
@@ -0,0 +1,57 @@
+package com.dotmarketing.business.cache.provider;
+
+import org.junit.Test;
+import com.dotcms.repackage.com.google.common.base.Optional;
+import com.dotmarketing.portlets.contentlet.business.Contentlet;
+
+public class CacheSizingUtilTest {
+
+    
+    final String fourLetters = "CRAP";
```

Comment:
Codacy found an issue: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6315945)